lychee-lib/src/types/input.rs

```diff
@@ -132,38 +132,59 @@ impl Input {
     ) -> Result<Self> {
         let source = if value == STDIN {
             InputSource::Stdin
-        } else if let Ok(url) = Url::parse(value) {
-            InputSource::RemoteUrl(Box::new(url))
+        // The extra use of Request::builder is to weed out Windows filepaths with a drive specifier, which Url will erroneously parse successfully
+        // We still use Url::parse because it catches some other edge cases that Request::builder does not
+        // This could be improved with further refinement.
```

Comment:
I'm guessing this comment is no longer needed? At least it looks like we don't use `Request::builder` anymore.

Reply:
removed the comment. let me know if it's still needed.